src/Components/Header/Header.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import Title from '../Shared/Title';
+import Nav from './Nav';
+import Search from './Search';
+
+import './Header.scss';
+
+export default function Header() {
```

Comment:
Don't forget to add ts type for all components and props

Reply:
but it don't expect any props, so 'props: {}' will be useless, isn't it?